ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/clientlibs/js/cartproductdisplay.js

```diff
@@ -123,6 +123,7 @@ localStorage.setItem('productDescription', JSON.stringify(productDescription ))
   };
  	xhttp.open("GET", "/bin/hclecomm/updateCartItems?payload=" +  JSON.stringify(cartItem) , true);
     xhttp.setRequestHeader("Content-Type", "application/json;charset=UTF-8");
+	  xhttp.setRequestHeader("CustomerToken", custToken);
```

Comment:
@DeepaliAgrwl : where is the assignation for  custToken done

Reply:
It Is given on top of the JS file inside document.ready {} function
Like This:
let userData = getUserCookie("hcluser"); 
    if(userData != "") {
         custToken = JSON.parse(userData).customerToken;
    }